doc/sphinx/conf.py

```diff
@@ -188,13 +188,66 @@ def setup(app):
     # but is it the right place?
     'module',
     'redexpr',
+    'red_expr',
     'modpath',
     'dirpath',
     'collection',
     'term_pattern',
     'term_pattern_string',
     'command',
-    'symbol' ]]
+    'symbol',
+
+    # added for Ltac chapter
+    'tacarg',
+    'ssr_first_else',
+    'ssrseqarg',
+    'int_or_var',
+    'string_opt',
+    'ssrmmod',
+    'ssrdotac',
+    'ssrclauses',
+    'ssrortacarg',
+    'ssrdgens',
+    'query_command',
+    'int_or_var_opt',
+    'simple_tactic',
+    'reference',
+    'ssrintros_ne',
+    'STRING',
+    'lconstr',
+    'uconstr',
+    'ssrparentacarg',
+    'lconstr_pattern',
+    'component',
+    'test',
+
+    # added for gallina chapter
+    'arg',
+    'binders',
+    'by_notation',
+    'cofix_body',
```

Comment:
Same thing for `cofix_body`.

Reply:
Only defined in the old version.

Reply:
Please do substitute `cofix_body` with `fix_body` everywhere it is used in the manual.